### PR TITLE
[webportal] job list paging

### PR DESCRIPTION
**Pull request by yiyione**

### CODE REVIEW

File: src/webportal/src/app/job/job-view/fabric/JobList/index.jsx [hunk 1]
```diff
@@ -136,12 +178,28 @@ export default function JobList() {
       token: token,
       https: window.location.protocol === 'https:',
     });
-    client.job
-      .listJobs()
+
+    const url = `${client.cluster.rest_server_uri}/api/v2/jobs`;
+    console.log(JSON.stringify(query));
```
Comment (debuggy) [inline src/webportal/src/app/job/job-view/fabric/JobList/index.jsx:183]: delete console.log ?

File: src/webportal/src/app/job/job-view/fabric/JobList/index.jsx [hunk 2]
```diff
@@ -136,12 +178,28 @@ export default function JobList() {
...
+    try {
+      return await client.httpClient.get(url, undefined, undefined, query);
+    } catch (err) {
+      throw Error(err.data.message || err.message);
```
Comment (debuggy) [inline src/webportal/src/app/job/job-view/fabric/JobList/index.jsx:187]: alert detailed error message before throw?

File: src/webportal/src/app/home/home.jsx [hunk 3]
```diff
@@ -47,7 +48,8 @@ import t from '../components/tachyons.scss';
 
 const Home = () => {
   const [loading, setLoading] = useState(true);
-  const [jobs, setJobs] = useState(null);
+  const [jobsStatusNumber, setJobsStatusNumber] = useState(null);
```
[the remark this answers is not in the document]
Comment (yiyione) [reply src/webportal/src/app/home/home.jsx]: Done.

File: src/webportal/src/app/job/job-view/fabric/JobList/index.jsx (hunk 1 above)
Comment (yiyione) [reply src/webportal/src/app/job/job-view/fabric/JobList/index.jsx]: Done.

File: src/webportal/src/app/job/job-view/fabric/JobList/index.jsx (hunk 2 above)
Comment (yiyione) [reply src/webportal/src/app/job/job-view/fabric/JobList/index.jsx:190]: Done.

**Review by debuggy — APPROVED** [review]

**Pull request merged**